src/sentry/conf/server.py

```diff
@@ -487,6 +487,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue('events.reprocess_events', routing_key='events.reprocess_events'),
     Queue('events.save_event', routing_key='events.save_event'),
     Queue('files.delete', routing_key='files.delete'),
+    Queue('incidents.notify', routing_key='incidents.notify'),
```

Comment:
Does operations need to know about new queues being created?

Reply:
Not sure, cc @JTCunning

Reply:
No, but it never hurts to check in. <3